extensions/wikia/GlobalFooter/GlobalFooterController.class.php

```diff
@@ -57,12 +56,19 @@ private function getGlobalFooterLinks() {
 			$link = trim( $link );
 			if ( strpos( $link, '*' ) === 0 ) {
 				$parsedLink = parseItem( $link );
-				if ( ( strpos( $parsedLink['text'], 'LICENSE' ) !== false ) || $parsedLink['text'] == 'GFDL' ) {
-					$parsedLink['isLicense'] = true;
+
+				if ( strpos( $parsedLink['text'], 'SITEMAP' ) !== false ) {
+
+					$parsedLinks = array_merge( $parsedLinks, $this->generateSitemapLinks() );
```

Comment:
Well, we don't want to overwrite the previous links already in parsedLinks, thus array_merge :-)